src/parser/shared/modules/StatTracker.ts

```diff
@@ -956,8 +974,10 @@ class StatTracker extends Analyzer {
             ' ...unable to handle stats buff, making no stat change.',
           );
         }
+        return 0;
       }
-      return 0;
+
+      return buffVal(selectedCombatant, null as any);
```

Comment:
`null as any` is a spicy one. this also makes the typing of the function unsound. `BuffVal` is a number or `(Combatant, Item) => number` and as a result can assume that the 2nd param is non-null since TS values are not inhabited by `null` by default.

this change should either be reverted or the type of `BuffVal` should be changed.

Reply:
fixed by allowing `null` for the `item` param and refining the behavior of the relevant functions